models/defaults.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface DefaultModelAttrs {
```

Comment:
I tried for a while to find a type inside of sequelize that adds the createdAt/updatedAt/id properties to objects returned from the database, but couldn't find one. So I ended up creating this type.